.exp-insertions.yml

```diff
@@ -182,19 +186,26 @@ jobs:
     displayName: Download msbuild vs drop
     env:
       cloudbuild-token: $(cloudbuild-token)
-      
+
   - task: DownloadBuildArtifacts@1
     inputs:
       buildType: specific
       project: DevDiv
-      pipeline: $(_MsBuildCiPipelineId) 
-      ${{ if eq(parameters.MSBuildBuildID, 'default') }}: 
+      pipeline: $(_MsBuildCiPipelineId)
+      ${{ if eq(parameters.MSBuildBuildID, 'default') }}:
         buildVersionToDownload: latestFromBranch
         branchName: '${{parameters.MSBuildBranch}}'
       ${{ else }}:
         buildVersionToDownload: specific
-        buildId: ${{parameters.MSBuildBuildID}} 
+        buildId: ${{parameters.MSBuildBuildID}}
       artifactName: bin
+      itemPattern: |
```

Comment:
Pattern seems fine for Framework MSBuild dogfooding, so this place is good.

There is one more place where pipeline downloads from `_MsBuildCiPipelineId`. We need filtering pattern there as well. As far as I know, `MSBuild.Bootstrap` is the only directory from bin that is used to update the experimental dotnet MSBuild dogfooding, so the pattern there should be even easier.

Reply:
Yeah, Line 81 is for dotnet filter

Reply:
Right, sorry, did not notice that line.